src/pytorch_lightning/callbacks/finetuning.py

```diff
@@ -290,18 +290,13 @@ def _store(
 
     def on_train_epoch_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule") -> None:
         """Called when the epoch begins."""
-        # import is here to avoid circular imports
-        from pytorch_lightning.loops.utilities import _get_active_optimizers
-
-        for opt_idx, optimizer in _get_active_optimizers(trainer.optimizers, trainer.optimizer_frequencies, 0):
+        for opt_idx, optimizer in enumerate(trainer.optimizers):
```

Comment:
This loop is left for manual optimization? Do we have tests covering manual with this callback?

Reply:
There isn't a test for it with manual optimization, but it should work because all the callback does is freeze and unfreeze parameters, which the user (who implements the finetune function) could do on a per-optimizer param group level.